src/common/dav/heap.c

```diff
@@ -1,5 +1,5 @@
 /* SPDX-License-Identifier: BSD-3-Clause */
-/* Copyright 2015-2023, Intel Corporation */
+/* Copyright 2015-2022, Intel Corporation */
```

Comment:
```suggestion
/* Copyright 2015-2023, Intel Corporation */
```

I won't comment on any more erroneous copyright date changes but please review all of your changes for this and correct.